README.rst

```diff
@@ -90,3 +90,9 @@ Testing checklist
 * If you are writing a Rule, add the test cases for your rule to ``specCases`` in ``test/spec/rule_spec.js``.
 
 * If needed, any new integration tests can go in ``test/spec/integration_spec.js``.
+
+
+Publishing a New Version
+========================
+
+When you have merged changes, update the version number in ``package.json``, and publish the new version to npm using ``npm publish``.
```

Comment:
I don't care whether the version bump is with the original PR, and I'm working on the assumption that you need thumbs on any PR. I'll add the note about NPM permissions. Honestly, I don't understand your last bullet.